django/applications/catmaid/models.py

```diff
@@ -1710,10 +1710,10 @@ def is_valid(self) -> bool:
             # The action is required to set a value for the is_valid variable.
             try:
                 _locals:Dict = {}
-                exec(self.validate_action, globals(), _locals)
+                exec(self.validate_action, globals(), _locals) # This directly manipulates Python's scope; type checkers will be angry.
```

Comment:
I agree, this logic is really old and also not the way I would prefer to do handle change request actions. Thanks for adding a comment there. Would you mind prefixing this with a `FIXME: `?